### DISCUSSION TITLE
[Announcement] The deadline has been extended by one day

### DISCUSSION THREAD

**Discussion by yeonhee-ryou** [prosyslab-classroom/cs348-information-security#458] [Announcements]
Due to the KCLOUD VPN issue, we have extended the homework 3 deadline by one day.
Also, we will provide temporal accounts through KLMS as soon as possible.

KCLOUD VPN 접근 이슈로 숙제 3 제출 기한을 하루 연장합니다.
임시 계정이 준비되는 대로 KLMS 를 통해 배포할 예정이니 참고 바랍니다.

- Extended Due: 04. 26 (Wed.) 23:59:59
- Late Due: 04. 28 (Fri.) 23:59:59 (late submission penalty).

**Comment by sujin0529** [comment]
We uploaded temporal accounts on KLMS.
Please check KLMS notice.

Thanks.
